backend/Tasque.Core.WebAPI/appsettings.json

```diff
@@ -12,9 +12,6 @@
     "Key": "BSA-2022-Tasque.Core"
   },
   "AllowedHosts": "*",
-  "ConnectionStrings": {
```

Comment:
Why is this line removed?

Reply:
We do not really need it, as we store this data in user's secrets. But I can return it if you want.